UI/EnemyDetailsPanel.gd

```diff
@@ -66,6 +66,8 @@ func _on_enemy_changed():
 		$VBoxContainer/GridContainer/HighOrbsContainer/HighOrbsSpinBox.value = enemy_clone.high_orbs
 		$VBoxContainer/ExpContainer/NamedParamsExpPercentageLabel.text = String(enemy_clone.named_param.experience)
 		$VBoxContainer/ExpContainer/ExpSpinBox.value = enemy_clone.experience
+		$VBoxContainer/PpContainer/NamedParamsPpPercentageLabel.text = String("100")
```

Comment:
Inconditionally 100%?

Reply:
Apologies - can you elaborate on what you mean by this? What is the problem?

Reply:
This has been fixed. It now varies like the experience text does.